3-networks-dual-svpc/modules/base_env/remote.tf

```
@@ -16,15 +16,18 @@
 
 locals {
   restricted_project_id        = data.terraform_remote_state.org.outputs.shared_vpc_projects[var.env].restricted_shared_vpc_project_id
-  restricted_project_number    = data.terraform_remote_state.org.outputs.shared_vpc_projects[var.env].restricted_shared_vpc_project_number
   base_project_id              = data.terraform_remote_state.org.outputs.shared_vpc_projects[var.env].base_shared_vpc_project_id
+  restricted_project_number    = data.terraform_remote_state.org.outputs.shared_vpc_projects[var.env].restricted_shared_vpc_project_number
   interconnect_project_number  = data.terraform_remote_state.org.outputs.interconnect_project_number
-  dns_hub_project_id           = data.terraform_remote_state.org.outputs.dns_hub_project_id
   organization_service_account = data.terraform_remote_state.bootstrap.outputs.organization_step_terraform_service_account_email
   networks_service_account     = data.terraform_remote_state.bootstrap.outputs.networks_step_terraform_service_account_email
   projects_service_account     = data.terraform_remote_state.bootstrap.outputs.projects_step_terraform_service_account_email
+  prod_restricted_project_id   = data.terraform_remote_state.org.outputs.shared_vpc_projects["production"].restricted_shared_vpc_project_id
+  prod_base_project_id         = data.terraform_remote_state.org.outputs.shared_vpc_projects["production"].base_shared_vpc_project_id
```

For consistency:

- change `prod_base_project_id` to `production_project_id`
or
- change `production_project_id` variable to `prod_base_project_id`

```suggestion
  prod_base_project_id         = data.terraform_remote_state.org.outputs.shared_vpc_projects["production"].base_shared_vpc_project_id
```